fs/vfs/fs_stat.c

```diff
@@ -303,6 +303,18 @@ int inode_stat(FAR struct inode *inode, FAR struct stat *buf)
           buf->st_mode  = S_IFMTD;
           buf->st_mode |= S_IROTH | S_IRGRP | S_IRUSR;
           buf->st_mode |= S_IWOTH | S_IWGRP | S_IWUSR;
+        
```

Comment:
Let's squash the 2nd path into 1st one:
git rebase --interactive HEAD~2
change pick to squash